src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/ExprFactory.cs

```diff
@@ -103,10 +87,7 @@ public ExprProperty CreateProperty(CType pType, Expr pOptionalObject)
 
         public ExprProperty CreateProperty(CType pType, Expr pOptionalObjectThrough, Expr pOptionalArguments, ExprMemberGroup pMemberGroup, PropWithType pwtSlot, MethWithType mwtGet, MethWithType mwtSet)
```

Comment:
`mwtGet` isn't used, but I didn't clean up any unused parameters that are non-trivially determined in a call, because until I'm 100% sure their not being used isn't a bug it'd be making it harder to fix rather than easier to analyse.